### PR TITLE
[BULK] - DocuTune remediation - Sensitive terms with GUIDs (part 20)

### PR DESCRIPTION
**Pull request by meganbradley**
Applying sensitive terms with GUID changes as part of Content SFI and outlined in [Overview - Writing content securely - Platform Manual](https://review.learn.microsoft.com/en-us/help/platform/security-reference?branch=main). Changes are part of the Microsoft-wide SFI effort. Point of contact: @CelesteDG

DocuTune v1.5.2.0
CorrelationId: [ac15aa43-4e2b-437f-ab1c-fdd7e79cd4db](https://github.com/Azure/azure-docs-powershell-azuread/pulls?q=is%3Apr+ac15aa43-4e2b-437f-ab1c-fdd7e79cd4db+)

#docutune

### CODE REVIEW

**Comment by meganbradley** [comment]
#docutune-review

File: azureadps-2.0/AzureAD/Set-AzureADMSPermissionGrantConditionSet.md [hunk 1]
```diff
@@ -31,9 +31,9 @@ Updates an Azure Active Directory permission grant condition set object identifi
 ```
 1. Get existing permission grant policy by that need to be updated.
 		
-		$permissionGrantConditionSet =Get-AzureADMSPermissionGrantConditionSet -PolicyId "test1" -ConditionSetType "includes" -Id "0f81cce0-a766-4db6-a7e2-4e5f10f6abf8"
+		$permissionGrantConditionSet =Get-AzureADMSPermissionGrantConditionSet -PolicyId "test1" -ConditionSetType "includes" -Id "1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5"
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Set-AzureADMSPermissionGrantConditionSet.md:34]: Not related to policy Id, needs reverting
```suggestion
		$permissionGrantConditionSet =Get-AzureADMSPermissionGrantConditionSet -PolicyId "test1" -ConditionSetType "includes" -Id "1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5"
```

File: azureadps-2.0/AzureAD/Set-AzureADMSPermissionGrantConditionSet.md [hunk 2]
```diff
@@ -31,9 +31,9 @@ Updates an Azure Active Directory permission grant condition set object identifi
...
 
-			Id                                          : 0f81cce0-a766-4db6-a7e2-4e5f10f6abf8
+			Id                                          : 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Set-AzureADMSPermissionGrantConditionSet.md:36]: As above
```suggestion
			Id                                          : 0f81cce0-a766-4db6-a7e2-4e5f10f6abf8
```

File: azureadps-2.0/AzureAD/Set-AzureADMSPermissionGrantConditionSet.md [hunk 3]
```diff
@@ -49,7 +49,7 @@ Updates an Azure Active Directory permission grant condition set object identifi
 		
 		Set-AzureADMSPermissionGrantConditionSet -PolicyId "test1" -ConditionSetType "includes" -Id $permissionGrantConditionSet.Id -PermissionClassification low
 
-			Id                                          : 0f81cce0-a766-4db6-a7e2-4e5f10f6abf8
+			Id                                          : 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Set-AzureADMSPermissionGrantConditionSet.md:52]:
```suggestion
			Id                                          : 0f81cce0-a766-4db6-a7e2-4e5f10f6abf8
```

File: azureadps-2.0/AzureAD/Set-AzureADMSPermissionGrantConditionSet.md [hunk 4]
```diff
@@ -64,7 +64,7 @@ Updates an Azure Active Directory permission grant condition set object identifi
 
 ### Example 2: Update a permission grant condition set
 ```
-PS C:\>Set-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "includes" -Id "665a9903-0398-48ab-b4e9-7a570d468b66" -PermissionType "Delegated" -PermissionClassification "Low" -ResourceApplication "d9d40050-c784-4b56-a06d-477542a1cafc" -Permissions @("29bf4ca5-913e-427d-8a68-5890af945109") -ClientApplicationIds @("All") -ClientApplicationTenantIds @("All") -ClientApplicationPublisherIds @("All") -ClientApplicationsFromVerifiedPublisherOnly $true
+PS C:\>Set-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "includes" -Id "2bbbbbb2-3cc3-4dd4-5ee5-6ffffffffff6" -PermissionType "Delegated" -PermissionClassification "Low" -ResourceApplication "3cccccc3-4dd4-5ee5-6ff6-7aaaaaaaaaa7" -Permissions @("4dddddd4-5ee5-6ff6-7aa7-8bbbbbbbbbb8") -ClientApplicationIds @("All") -ClientApplicationTenantIds @("All") -ClientApplicationPublisherIds @("All") -ClientApplicationsFromVerifiedPublisherOnly $true
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Set-AzureADMSPermissionGrantConditionSet.md:67]:
```suggestion
PS C:\>Set-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "includes" -Id "665a9903-0398-48ab-b4e9-7a570d468b66" -PermissionType "Delegated" -PermissionClassification "Low" -ResourceApplication "00001111-aaaa-2222-bbbb-3333cccc4444" -Permissions @("29bf4ca5-913e-427d-8a68-5890af945109") -ClientApplicationIds @("All") -ClientApplicationTenantIds @("All") -ClientApplicationPublisherIds @("All") -ClientApplicationsFromVerifiedPublisherOnly $true
```

File: docs-conceptual/azureadps-2.0-preview/recovering-deleted-data.md [hunk 5]
```diff
@@ -55,11 +55,11 @@ The first cmdlet will retrieve the deleted unified groups in your directory, the
 If you want to recover a deleted application object, you can use
 
 ```powershell
-Restore-AzureADDeletedApplication -ObjectId 9a7d49c5-42ee-47a3-bad7-17cae5bb8f26
+Restore-AzureADDeletedApplication -ObjectId aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb
 
 ObjectId                             AppId                                DisplayName
 --------                             -----                                -----------
-9a7d49c5-42ee-47a3-bad7-17cae5bb8f26 7dba6cec-ffd5-40af-ba40-1eb158574b5a My Properties Bag
+aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb 7dba6cec-ffd5-40af-ba40-1eb158574b5a My Properties Bag
```
Comment (cilwerner) [inline docs-conceptual/azureadps-2.0-preview/recovering-deleted-data.md:62]:
```suggestion
aaaaaaaa-0000-1111-2222-bbbbbbbbbbbb 00001111-aaaa-2222-bbbb-3333cccc4444 My Properties Bag
```

**Review by cilwerner — CHANGES_REQUESTED** [review]
@msewaweru, I've made a few suggestions here. We can commit these and merge, if you're able to do it

File: azureadps-2.0/AzureAD/Set-AzureADMSNamedLocationPolicy.yml [hunk 6]
```diff
@@ -17,13 +17,13 @@ syntaxes:
 examples:
 - title: 'Example 1: Update an ip named location policy in Azure AD by PolicyId.'
   code: |-
-    PS C:\> Set-AzureADMSNamedLocationPolicy -PolicyId 07a1f48d-0cbb-4c2c-8ea2-1ea00e3eb3b6 -OdataType "#microsoft.graph.ipNamedLocation" -IsTrusted $false
+    PS C:\> Set-AzureADMSNamedLocationPolicy -PolicyId 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5 -OdataType "#microsoft.graph.ipNamedLocation" -IsTrusted $false
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Set-AzureADMSNamedLocationPolicy.yml:20]: Policy id now excluded from sensitive term list
```suggestion
    PS C:\> Set-AzureADMSNamedLocationPolicy -PolicyId 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5 -OdataType "#microsoft.graph.ipNamedLocation" -IsTrusted $false
```

File: azureadps-2.0/AzureAD/Set-AzureADMSNamedLocationPolicy.yml [hunk 7]
```diff
@@ -17,13 +17,13 @@ syntaxes:
...
   description: |-
     This command updates an ip named location policy in Azure AD by PolicyId.
   summary: ""
 - title: 'Example 2: Update a country or region named location policy in Azure AD by PolicyId.'
   code: |-
-    PS C:\> Set-AzureADMSNamedLocationPolicy -PolicyId 76fdfd4d-bd80-4c1e-8fd4-6abf49d121fe -OdataType "#microsoft.graph.countryNamedLocation" -IncludeUnknownCountriesAndRegions $true
+    PS C:\> Set-AzureADMSNamedLocationPolicy -PolicyId 2bbbbbb2-3cc3-4dd4-5ee5-6ffffffffff6 -OdataType "#microsoft.graph.countryNamedLocation" -IncludeUnknownCountriesAndRegions $true
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Set-AzureADMSNamedLocationPolicy.yml:26]:
```suggestion
    PS C:\> Set-AzureADMSNamedLocationPolicy -PolicyId 76fdfd4d-bd80-4c1e-8fd4-6abf49d121fe -OdataType "#microsoft.graph.countryNamedLocation" -IncludeUnknownCountriesAndRegions $true
```

File: azureadps-2.0/AzureAD/Set-AzureADMSPermissionGrantConditionSet.yml [hunk 8]
```diff
@@ -30,9 +30,9 @@ examples:
   code: |-
     1. Get existing permission grant policy by that need to be updated.
     		
-    		$permissionGrantConditionSet =Get-AzureADMSPermissionGrantConditionSet -PolicyId "test1" -ConditionSetType "includes" -Id "0f81cce0-a766-4db6-a7e2-4e5f10f6abf8"
+    		$permissionGrantConditionSet =Get-AzureADMSPermissionGrantConditionSet -PolicyId "test1" -ConditionSetType "includes" -Id "1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5"
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Set-AzureADMSPermissionGrantConditionSet.yml:33]:
```suggestion
    		$permissionGrantConditionSet =Get-AzureADMSPermissionGrantConditionSet -PolicyId "test1" -ConditionSetType "includes" -Id "0f81cce0-a766-4db6-a7e2-4e5f10f6abf8"
```

File: azureadps-2.0/AzureAD/Set-AzureADMSPermissionGrantConditionSet.yml [hunk 9]
```diff
@@ -30,9 +30,9 @@ examples:
...
 
-    			Id                                          : 0f81cce0-a766-4db6-a7e2-4e5f10f6abf8
+    			Id                                          : 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Set-AzureADMSPermissionGrantConditionSet.yml:35]:
```suggestion
    			Id                                          : 0f81cce0-a766-4db6-a7e2-4e5f10f6abf8
```

File: azureadps-2.0/AzureAD/Set-AzureADMSPermissionGrantConditionSet.yml [hunk 10]
```diff
@@ -48,7 +48,7 @@ examples:
     		
     		Set-AzureADMSPermissionGrantConditionSet -PolicyId "test1" -ConditionSetType "includes" -Id $permissionGrantConditionSet.Id -PermissionClassification low
 
-    			Id                                          : 0f81cce0-a766-4db6-a7e2-4e5f10f6abf8
+    			Id                                          : 1aaaaaa1-2bb2-3cc3-4dd4-5eeeeeeeeee5
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Set-AzureADMSPermissionGrantConditionSet.yml:51]:
```suggestion
    			Id                                          : 0f81cce0-a766-4db6-a7e2-4e5f10f6abf8
```

File: azureadps-2.0/AzureAD/Set-AzureADMSPermissionGrantConditionSet.yml [hunk 11]
```diff
@@ -63,7 +63,7 @@ examples:
   summary: ""
 - title: 'Example 2: Update a permission grant condition set'
   code: |-
-    PS C:\>Set-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "includes" -Id "665a9903-0398-48ab-b4e9-7a570d468b66" -PermissionType "Delegated" -PermissionClassification "Low" -ResourceApplication "d9d40050-c784-4b56-a06d-477542a1cafc" -Permissions @("29bf4ca5-913e-427d-8a68-5890af945109") -ClientApplicationIds @("All") -ClientApplicationTenantIds @("All") -ClientApplicationPublisherIds @("All") -ClientApplicationsFromVerifiedPublisherOnly $true
+    PS C:\>Set-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "includes" -Id "2bbbbbb2-3cc3-4dd4-5ee5-6ffffffffff6" -PermissionType "Delegated" -PermissionClassification "Low" -ResourceApplication "3cccccc3-4dd4-5ee5-6ff6-7aaaaaaaaaa7" -Permissions @("4dddddd4-5ee5-6ff6-7aa7-8bbbbbbbbbb8") -ClientApplicationIds @("All") -ClientApplicationTenantIds @("All") -ClientApplicationPublisherIds @("All") -ClientApplicationsFromVerifiedPublisherOnly $true
```
Comment (cilwerner) [inline azureadps-2.0/AzureAD/Set-AzureADMSPermissionGrantConditionSet.yml:66]:
```suggestion
    PS C:\>Set-AzureADMSPermissionGrantConditionSet -PolicyId "policy1" -ConditionSetType "includes" -Id "665a9903-0398-48ab-b4e9-7a570d468b66" -PermissionType "Delegated" -PermissionClassification "Low" -ResourceApplication "d9d40050-c784-4b56-a06d-477542a1cafc" -Permissions @("29bf4ca5-913e-427d-8a68-5890af945109") -ClientApplicationIds @("All") -ClientApplicationTenantIds @("All") -ClientApplicationPublisherIds @("All") -ClientApplicationsFromVerifiedPublisherOnly $true
```

**Review by cilwerner — CHANGES_REQUESTED** [review]
@msewaweru, a couple of changes here, then good to go!

**Review by msewaweru — CHANGES_REQUESTED** [review]

**Review by msewaweru — APPROVED** [review]

**Pull request merged**